cub/cub/device/dispatch/tuning/tuning_transform.cuh

```diff
@@ -128,13 +222,23 @@ _CCCL_HOST_DEVICE constexpr auto bulk_copy_alignment(int ptx_version) -> int
   return ptx_version < 1000 ? 128 : 16;
 }
 
-template <typename... RandomAccessIteratorsIn>
-_CCCL_HOST_DEVICE constexpr auto bulk_copy_smem_for_tile_size(int tile_size, int bulk_copy_align) -> int
+template <typename ItValueSizesAlignments>
+_CCCL_HOST_DEVICE constexpr auto
+bulk_copy_smem_for_tile_size(ItValueSizesAlignments it_value_sizes_alignments, int tile_size, int bulk_copy_align)
+  -> int
 {
   return ::cuda::round_up(int{sizeof(int64_t)}, bulk_copy_align) /* bar */
        // 128 bytes of padding for each input tile (handles before + after)
-       + tile_size * loaded_bytes_per_iteration<RandomAccessIteratorsIn...>()
```

Comment:
Important: this was the last use of `loaded_bytes_per_iteration` so I think we can drop the entire function now (including from the `TransformKernelSource`).